README.md

```diff
@@ -115,6 +115,48 @@ jobs:
           sceptre_subcommand: 'launch -y dev'
 ```
 
+```yaml
+# Function: Run pre-commit checks and deploy 'dev' stack group if checks pass
+# Trigger: Push/Merge to all branches, but deployment reserved to 'main' branch
+# Plugins: Install from a Pipfile using the specified version of Pipenv 
+name: 'Sceptre Github CI Action'
+on: push
+jobs:
+  pre-commit:
+    name: 'Run pre-commit hooks against all files'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.2
```

Comment:
To make it less confusing I think we really just want the simplest possible example.  

Do we really need pre-commit in an example for sceptre GH action?